cache/src/convertable.rs

```diff
@@ -22,41 +21,50 @@ pub trait Convertable {
 	/// Generally, it fails if the accessible item does not implement to accessible interface.
 	/// This shouldn't be possible, but this function may fail for other reasons.
 	/// For example, to convert a [`zbus::Proxy`] into a [`Self::Accessible`], it may fail to create the new [`atspi_proxies::accessible::AccessibleProxy`].
-	async fn to_accessible(&self) -> Result<AccessibleProxy, Self::Error>;
+	fn to_accessible(
+		&self,
+	) -> impl Future<Output = Result<AccessibleProxy, Self::Error>> + Send;
```

Comment:
I mean, in stead of this:

```rust
	fn to_accessible(
 		&self,
 	) -> impl Future<Output = Result<AccessibleProxy, Self::Error>> + Send;
```

you could do something like this:

```rust
	async fn to_accessible(
 		&self,
 	) -> Result<AccessibleProxy, Self::Error>>;
```

I dk about accessible proxy, but essentially the send value shouldn't be required on the future itself, because the compiler boxes it, and futures are send by default unless otherwise noted, so yeah. I dk if that's possible to be done, but it will make for less a code complexity